### DISCUSSION TITLE
example of --key-cutoff-time

### DISCUSSION THREAD

**Discussion by shyam-ks** [bitnami-labs/sealed-secrets#925] [Q&A]
Please share an example of how to use --key-cutoff-time with its value to force rotate the key.
Thanks

**Comment by aladrocMatiner** [comment]
Hej, can u put a practical example on how to force rotate the keys?? is not super clear and on the version 0.19.3 a lot of flags that show on the documentation are missing.... so starts to become hard to follow what is happening in the project...

**Reply by aladrocMatiner** [reply]
this is defined in the controller not in the kubeseal that is the confusion... just go to the controller yaml and add it on the arguments:
```
          image: 'docker.io/bitnami/sealed-secrets-controller:v0.19.3'
          args:
            - '--key-renew-period=1h'
            - '--update-status'
            - '--key-prefix'
            - sealed-secrets-key
  ```
